lib/processor.js

```diff
@@ -29,9 +30,11 @@ function parse(text) {
         column = 0,
         mode = modes.NORMAL,
         next,
-        expErr;
+        expErr,
+        expErrQty;
 
-    var EXP_ERR_REGEX = /\/\*\s*error\s+(.+?)\s*\*\//g;
+    var EXP_ERR_REGEX = /\/\*\s*error\s+(.+?)\s*\*\//g,
+        EXP_ERR_QTY_REGEX = /\/\/\s*(.+?) errors\s*/;
```

Comment:
`\d+` inside the match group will only capture digits (right now `// Three errors` matches).

Is the goal that the `// x errors` form will only be used when there are at least two errors on a line and listing them individually would be too long? (i.e. no need to check for `// 1 error`)